docs/tutorials.md

```diff
@@ -1,9 +1,10 @@
 # Tutorials
 
-All tutorials are written in Google Colab so that you can easily follow them without any additional setup
+All tutorials are opened in Google Colab so that you can easily follow them without any additional setup. If you want you can always access [raw .ipynb tutorial files](https://github.com/deepsense-ai/db-ally/tree/main/docs/tutorials).
```

Comment:
I don't think this should be the first link on the page. I would move it to the bottom (or remove it altogether).

Reply:
removed